coalib/results/Result.py

```diff
@@ -35,7 +37,8 @@ def __init__(self,
                  severity: int=RESULT_SEVERITY.NORMAL,
                  additional_info: str="",
                  debug_msg="",
-                 diffs: (dict, None)=None):
+                 diffs: (dict, None)=None,
+                 confidence: int=100):
```

Comment:
float isn't precise, a percentage between 0 and 100 is definitely fine grained enough and simple to deal with without having to worry about precision issues with floats